common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -86,6 +87,9 @@ object Messages {
                                       device: Uuid,
                                       status: DeviceStatus) extends BusMessage
 
+  final case class CampaignLaunched(namespace: Namespace, updateId: Uuid, devices: Set[Uuid], pkgUri: Uri,
```

Comment:
Well he has the uri, but does he have the credentials to download it from s3 if it's not a signed url?